assets/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@use 'sass/01-generic/generic';
+@use 'sass/02-components/header';
+@use 'sass/02-components/scroll-indicator';
+@use 'sass/03-utils/header-scroll-animation';
```

Comment:
Can you import reset css from package reset-css?, I think that reset sass file is in diferent path on node_modules

Reply:
Yes sir